sem_covid/entrypoints/etl_dags/eurlex_covid19.py

```diff
@@ -508,16 +508,16 @@ def download_dataset_items(json_file_name):
         else:
             logger.exception(f"No manifestation has been found for {item['title']}")
 
-    minio.put_object_from_string(json_file_name, json.dumps(json_content))
+    minio.put_object(json_file_name, json.dumps(json_content))
```

Comment:
put_object then receives a path? or just a file name? 
this shall be clearly documented in the Adapter.